### PR TITLE
Growth engine updates

### PR DESCRIPTION
**Pull request by sparrowDom**
### Description:
Some updates to the growth engine: 
- Add DateTime scalar to graphQl
- Create boilerplate `getCampaigns` resolver
- add primary keys, enum definitions and table names to model files
- add models index.js file

### Checklist:

- [ ] Test your work and double-check to confirm that you didn't break anything
- [ ] Wrap any displayed ETH addresses with [`formattedAddress`](https://github.com/OriginProtocol/origin/blob/master/origin-dapp/src/utils/user.js#L15-L17)
- [ ] Wrap any new text/strings for translation
- [ ] Run `npm run translations` if there are any changes to translated strings
- [ ] Map any new environment variables with a default value in the Webpack config
- [ ] Update any relevant READMEs and [docs](https://github.com/OriginProtocol/origin/tree/master/origin-docs)

### CODE REVIEW

**Comment by sparrowDom** [comment]
@franckc was creating a getCampaigns resolver and needed to modify model files a bit. Just a heads up, so we won't both be implementing the same thing.

File: origin-growth/src/apollo/db.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+  return {
+    id: row.id,
+    name: row.name,
+    startDate: startDate,
+    endDate: endDate,
+    distributionDate: row.distributionDate,
+    status: status,
+    actions: [], // TODO implement 
+    rewardEarned: { currency: 'OGN', amount: 0 } // TODO implement
```
Comment (franckc) [inline origin-growth/src/apollo/db.js]: note that there is a campaign.currency field. so let's use that rather than hardcode OGN.

Comment (franckc) [reply origin-growth/src/apollo/db.js]: Also rewardEarned will be by user so it's probably a method that won't be implemented in db.js but in a different layer ? In work-in-progress PR #1359 I have a Campaign class capable of evaluating rules per user. So maybe it should end up there ?

File: experimental/origin-graphql/src/typeDefs/Growth.js [hunk 2]
```diff
@@ -127,6 +128,8 @@ module.exports =
   type Mutation {
     invite(emails: [String!]!): InviteResponse
     enroll(campaignId: Int!): EnrollResponse
-    log(event: JSON!): LogResponse
+    gasForIdentity(walletAddress: ID!): SimpleResponse
```
Comment (franckc) [inline experimental/origin-graphql/src/typeDefs/Growth.js]: Spec is not finalized but I think we may give users enough ETH to pay for gas not only for Identity but also to create a listing. So maybe we give this mutation a more generic name like "sponsoredGas" ?

File: origin-growth/src/apollo/db.js (hunk 1 above)
Comment (sparrowDom) [reply origin-growth/src/apollo/db.js]: Good point removing that in a separate PR

File: experimental/origin-graphql/src/typeDefs/Growth.js (hunk 2 above)
Comment (sparrowDom) [reply experimental/origin-graphql/src/typeDefs/Growth.js]: Good point 👍

**Pull request merged**